.github/actions/push-image-aws-azure-gcp/action.yml

```diff
@@ -52,6 +64,32 @@ runs:
         docker tag $IMAGE:$TAG $REGISTRY/$REPOSITORY:$TAG
         docker push $REGISTRY/$REPOSITORY:$TAG
 
+    - name: Authenticate to GCP
+      uses: google-github-actions/auth@v1
+      with:
```

Comment:
Nerozmyslali ste, aby sme mali "niekde" jeden register, a pushovalo by sa iba tam?
A "nejako" by sa to automaticky rozdistribuovalo do vsetkych cloud providers?

Asi ste rozmyslali, len mi pride kopec zbytocnej manualnej prace to pridavat do vsetkych services.
Mozno keby to bola zdielana Github Action, tak by to bolo jednoduchsie.

Reply:
jo premysleli ale zatim to delame takhle

Reply:
ano, nyní jdeme nejrychlejší cestou, protože není moc času